lib/glob.c

```diff
@@ -109,7 +110,9 @@ EXPORTED glob *glob_init(const char *str, char sep)
     buf_appendcstr(&buf, "]|$)");
 
     glob *g = xmalloc(sizeof(glob));
-    regcomp(&g->regex, buf_cstring(&buf), REG_EXTENDED);
+    r = regcomp(&g->regex, buf_cstring(&buf), REG_EXTENDED);
+    /* XXX handle regex compilation failure properly! */
+    assert(r == 0);
```

Comment:
```
lib/glob.c: In function 'glob_init':
lib/glob.c:115:5: error: implicit declaration of function 'assert' [-Werror=implicit-function-declaration]
  115 |     assert(r == 0);
      |     ^~~~~~
lib/glob.c:51:1: note: 'assert' is defined in header '<assert.h>'; did you forget to '#include <assert.h>'?
```

Reply:
Oh good catch, thanks!  I'll get a fix for that up shortly

Reply:
Fixed in #4749